topics/ecology/tutorials/Metashrimps_tutorial/tutorial.md

```diff
@@ -0,0 +1,134 @@
+>
+>
+><img src="./Images/FAIR_data_principles.jpg" alt="FAIR Data Principles" width="500"/>
+
+>These principles were [officialy instaured in 2016](https://doi.org/10.1038/sdata.2016.18) to improve the access and usabiliy of data by the machine and to help making data reusable and shareable for users.
+>Metadata is the data used to describe and explain all the context behind the production of data. It is necessary to produce a rich and FAIR metadata in order 
+>to permit external users to understand and reuse data for their own studies.
+{:  .comment}
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+# How can this tool improve the metadata quality?
+
+The purpose of this tool is to help the user improve their metadata quality in order to increase its value to the scientific community and to help highlighting
+the work of all the producers of the data.
+To respond to this objective, this tool aims to give an easy access to a quality assessment report of EML metadata which could guide the producers of
+data/metadata to the production of a metadata with high degree of FAIRness.
+The other objective of this tool is to highlight the work of all of the people that helped in producing this data by giving the access of a draft of Data Paper
+that is reuseable either in a non editable HTML file, that can represent the metadata in a more ergonomic way facilitating its understanding and shareability,
+or in an editable docx file. Having an editable Data Paper draft will allow the producer to complete/modify the draft of Data Paper so that it could become
+publishable as a real Data Paper giving recognition to all the people that helped producing the data.
+
+> <comment-title>What is a Data Paper?</comment-title>
+> According to the [GBIF](https://www.gbif.org/data-papers) (Global Biodiversity Information Facility), 
+>A data paper is a peer reviewed document describing a dataset, published in a peer reviewed journal. It takes effort to prepare, curate and describe data. 
+>Data papers provide recognition for this effort by means of a scholarly article.
+{:  .comment}
+# Get data
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial
+> 2.Import this metadata file from [Zenodo]({{ https://doi.org/10.5281/zenodo.8130567 }}) to test it
+>     -> `{{ Training Data for "Creating Quality FAIR assessment reports and draft of Data Papers from EML metadata with MetaShRIMPS" }}`):
+>    ```
+>    https://zenodo.org/record/8130567/files/Kakila_database_marine_mammal.xml
+>    ```
+>
+>    {% snippet topics/ecology/tutorials/Metashrimps_tutorial/import_files.md %}
+
+# Upload Data in MetaShRIMPS
+
+When oppening MetaShRIMPS, you will have an interface looking like this :
+
+<img src="./Images/upload_1.png" alt="FAIR Data Principles" width="500"/>
```

Comment:
```suggestion
![FAIR Data Principles](./Images/upload_1.png){:width="500"}
```

you can write it like this for markdown

Reply:
It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details. *This comment may apply to more images, please consider checking all of them.*